src/controllers/update/update.director.check.answers.controller.ts

```diff
@@ -62,6 +62,7 @@ const renderPage = async (req: Request, res: Response, companyNumber: string, of
     nationalityLink: urlUtils.getUrlToPath("", req),
     occupationLink: urlUtils.getUrlToPath("", req),
     dateUpdatedLink: urlUtils.getUrlToPath("", req),
-    correspondenceAddressChangeLink: urlUtils.getUrlToPath("", req)
+    correspondenceAddressChangeLink: urlUtils.getUrlToPath("", req),
+    currentUrl: req.originalUrl,
```

Comment:
What is this currentUrl parameter doing?

Reply:
adds a language banner on the CYA page when `LOCALES_ENABLED` is enabled, without it I ran into the "Something went wrong" page.